### DISCUSSION TITLE
How to import CSV with less than all columns?

### DISCUSSION THREAD

**Discussion by slim186** [TheAxelander/OpenBudgeteer#217] [General]
Hello, my bank CSV files have 4 columns: date, description, debit, credit.  When trying to import, I cannot leave the Payee or Memo columns set to either description for both, or description for one and Select Column for the other.  It either says "Duplicate column mapping error" or "Index was outside the bounds of the array.".  Also my CSV files do not have a header row, do I have to add one every time or is there a way to specify "column 1", "column 2" etc?

**Comment by TheAxelander** [comment]
There is indeed some unintended behavior of the pre-selection for mapping the fields which is already fixed for the next version.

If your CSV file doesn't have any header row you would have to add one manually. The next version will then also support direct editing of the file on the Import Page (see #196)

For the final mapping if you set the Payee back to `---Select Column---` it should then normally work

![image](https://github.com/TheAxelander/OpenBudgeteer/assets/17849665/62b29ae2-5f82-4321-b850-1627e7b081ca)

**Reply by slim186** [reply]
No, I still get "Error message: One or more errors occurred. (Index was outside the bounds of the array.)" when trying to Validate Data after setting Payee to `---Select Column---`.  I'm using 1.7.1 (latest)

**Reply by TheAxelander** [reply]
Strange, did you add a header row? How does your selection looks like?
